### ISSUE TITLE
emphasized selected action-button focus ring is not visible. 

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-css#1172]
## Description
Focus ring for emphasized action button is barely visible and does not conform to spectrum

## Steps to reproduce

1. Go to http://opensource.adobe.com/spectrum-css/
2. Click on Action Button
3. Tab to the emphasized selected action buttons

## Expected behavior
Should meet spectrum 

## Screenshots
Spectrum screenshot 

![action-button_behaviors_keyboard-focus@2x_6Bps7XXN64M4s6rznKdMUd_1611634224000](https://user-images.githubusercontent.com/1724479/115648956-3f642f80-a2db-11eb-8138-18136f1af886.png)

Observed Behaviour - the first edit button has focus

<img width="143" alt="Screen Shot 2021-04-21 at 7 48 01 PM" src="https://user-images.githubusercontent.com/1724479/115648574-843b9680-a2da-11eb-926a-f3fbf8b978e3.png">

## Environment
 - **Component version:** @spectrum-css/actionbutton@1.0.0
 - **Browser(s) and OS(s):** Edge 90.0.818.42 on MacOs 11.2.3

**Comment by lazd** [comment]
Duplicated by #1177 with more information, closing this one!
